tutorial/solutions/Holmakefile

```diff
@@ -2,18 +2,12 @@ INCLUDES = $(HOLDIR)/examples/balanced_bst ../../misc ../../translator ../../bas
 					 ../../semantics ../../semantics/proofs ../../compiler/backend/proofs ../../compiler/backend/x64/proofs
 CLINE_OPTIONS = --qof
 
-SOLUTIONS = wordfreqProgScript.sml simple_bstScript.sml
-
 THYFILES = $(patsubst %Script.sml,%Theory.uo,$(wildcard *.sml))
 TARGETS0 = $(patsubst %Theory.sml,,$(THYFILES))
 TARGETS = $(patsubst %.sml,%.uo,$(TARGETS0))
 
-all: $(TARGETS) exercises
+all: $(TARGETS)
 
 .PHONY: all
 
-exercises: $(SOLUTIONS) make_ex
-	./make_ex $(SOLUTIONS)
-
-make_ex: make_ex.sml
-	$(POLYC) make_ex.sml -o make_ex
+# To build the exercises from the solutions, run `Holmake exercises` from `../../developers`
```

Comment:
No, I think this should be an automatic target here. (Or at least an automatic target somewhere.)

Reply:
Oh yeah, an automatic target is a good idea. What if `make_ex` lived in a subdirectory of `developers`, with a dependency on the bootstrapped compiler. Could pave the way to writing a few more utilities in concrete syntax?

Reply:
Yep.